app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AlbumsController < ApplicationController
+  before_action :get_album, only: [:show, :edit, :destroy, :upvote, :update]
+
+  def get_album
```

Comment:
Traditionally all methods used as `before_action`s are defined as `private`, along with the strong params methods.